at-tanvo/test.rb

```diff
@@ -0,0 +1,5 @@
+class WelcomeController < ApplicationController
+  def index
+  	#This line is too longgggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggg
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/2aac546f541d28724e2b45aad4eaaf26fa321660/at-tanvo/test.rb#L3"></span>Layout/CommentIndentation: Incorrect indentation detected (column 3 instead of 4).</td>
    </tr>
  </tbody>
</table>